ucan-invocation/tables/w3-metrics.js

```diff
@@ -0,0 +1,58 @@
+import {
+  DynamoDBClient,
+  UpdateItemCommand
+} from '@aws-sdk/client-dynamodb'
+import { marshall } from '@aws-sdk/util-dynamodb'
+
+import { W3_METRICS_NAMES } from '../constants.js'
+
+/**
+ * @typedef {import('@ucanto/interface').Ability} Ability
+ * @typedef {import('@ucanto/interface').Capability<Ability, `${string}:${string}`, unknown>[]} Capabilities
+ * 
+ * @typedef {object} UpdateInput
+ * @property {`did:${string}:${string}`} space
+ * @property {number} count
+ */
+
+/**
+ * Abstraction layer to handle operations on w3 metrics Table.
+ *
+ * @param {string} region
+ * @param {string} tableName
+ * @param {object} [options]
+ * @param {string} [options.endpoint]
+ * @returns {import('../types').W3MetricsTable}
+ */
+export function createW3MetricsTable (region, tableName, options = {}) {
+  const dynamoDb = new DynamoDBClient({
+    region,
+    endpoint: options.endpoint
+  })
+
+  return {
+    /**
+     * Increment accumulated value from new given operations.
+     *
+     * @param {Capabilities} operationsInv
+     */
+    incrementAccumulatedSize: async (operationsInv) => {
```

Comment:
This should either be specific to store add i.e. `incrementStoreAddAccumulatedSize` or take the key as a param.